contrib/database/sql/conn.go

```diff
@@ -146,6 +146,13 @@ func (tc *tracedConn) CheckNamedValue(value *driver.NamedValue) error {
 	return driver.ErrSkip
 }
 
```

Comment:
Maybe it makes sense to also add
```suggestion
var _ driver.SessionResetter = (*tracedConn)(nil)
```